integration_test_lite.py

```diff
@@ -145,16 +148,29 @@ async def get_bad_apis(self):
         print('Passing cases:')
         # Execute all tests in parallel
         results = await asyncio.gather(
-            *[self.bad_response(endpoint) for endpoint in endpoints]
+            *[self.test_endpoint(endpoint) for endpoint in endpoints]
         )
-        # Return all results that weren't None
-        return [result for result in results if result]
+
+        good_apis = [result for result in results if result['response_code'] == 200]
+        bad_apis = [result for result in results if result['response_code'] != 200]
+
+        return good_apis, bad_apis
+
+
+def write_log(good_apis, bad_apis):
+    log = {
```

Comment:
Hmm, so why not directly return this object in the first place (`get_api_status`)? So that you don't need to carry `good_apis` and `bad_apis` all the time?